Description: Upstream changes introduced in version 1.4-5nexenta1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 tmux (1.4-5nexenta1) unstable; urgency=low
 .
   * NCP4 port
 .
 The person named in the Author field signed this changelog entry.
Author: Alexander Eremin <a.eremin@nexenta.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tmux-1.4.orig/Makefile
+++ tmux-1.4/Makefile
@@ -50,7 +50,7 @@ CFLAGS+= -Wno-pointer-sign
 PREFIX?= /usr/
 INSTALL?= install
 INSTALLDIR= ${INSTALL} -d
-INSTALLBIN= $(INSTALL) -g utmp -o root -m 2755
+INSTALLBIN= $(INSTALL) -g root -o root -m 2755
 INSTALLMAN= ${INSTALL} -m 444
 
 SRCS!= echo *.c|LC_ALL=C sed 's|osdep-[a-z0-9]*.c||g'
--- tmux-1.4.orig/GNUmakefile
+++ tmux-1.4/GNUmakefile
@@ -54,7 +54,7 @@ endif
 PREFIX?= /usr/
 INSTALL?= install
 INSTALLDIR= $(INSTALL) -d
-INSTALLBIN= $(INSTALL) -g utmp -o root -m 2755
+INSTALLBIN= $(INSTALL) -g root -o root -m 2755
 INSTALLMAN= $(INSTALL) -m 444
 
 SRCS= $(shell echo *.c|LC_ALL=C sed 's|osdep-[a-z0-9]*.c||g')
--- tmux-1.4.orig/tmux.c
+++ tmux-1.4/tmux.c
@@ -195,7 +195,7 @@ makesocketpath(const char *label)
 		return (NULL);
 	}
 	/* drop unnecessary privileges */
- 	if (setresgid(gid, gid, gid) != 0)
+ 	if (setgid(gid) != 0)
  		return (NULL);
 
 	xasprintf(&path, "%s/%s", base, label);
